internal/conf/path.go

```diff
@@ -69,6 +85,16 @@ func GetPluginsLoc() (string, error) {
 
 func absolutePath(loc string) (dir string, err error) {
 	for relDir, absoluteDir := range AbsoluteMapping {
+		switch loc {
+		case etcDir:
```

Comment:
This is wrong. The loc could be somthing like "etc/sources/abc.yaml" not just "etc". The test case only covers when loc==etcDir case. ,

Previously, AbsoluteMapping is a const. Generally, we just need to let it point to the flag value if is explicitly set. No need to touch the logic here.

Reply:
updated.